### PR TITLE
Demo submission: Message brokering using RabbitMQ

### PR DESCRIPTION
**Pull request by iriediese**
Original PR: #1077  
Everything else is in the README.md

### CODE REVIEW

**Comment by arxra** [comment]
#Feedback 
Nice seeing some love for rabbitmq and LXD in the course, and very nice having the source code in the description.
Some suggestions:

- The text in the demo is really small, at many times without the need for such small text. Take 1:35 (timestamp) as example, most of the screen is taken up by download statuses instead of the functionality being demoed. 
- The voice over is quite good. There are just a few "uhm" in between the lines but its not too much in the viewers face.
- There is not a lot going on, the actual content seems to be quite short. There could be some time spent explaining what the worker-queue pattern is, what rabbitmq is, or a nicer way of getting the dependancies for the project.

While the demo was visually hard to follow, it was nicely layed out and easy to follow verbally.

**Comment by altaired** [comment]
# Overall

Good video, with a really interesting topic. I've always wanted to look into rabbitMQ.

# Strengths
- The topic is nice, with a clear target. 
- It has a calm tempo, compared to a lot of other videos that are way to fast.
- The subtitles are matching the voiceover properly

# Actionable points
- A really important improvement is the font size, the text is way to small. Sometimes it is very hard to see what you are typing
- Recommend you to rerecord the voiceover, since the current audio seems very conversational and not always to the point, a lot of "uhm"s.
- Would be nice with a bit more introduction to RabbitMQ.

**Comment by Spycsh** [comment]
**Good Job!**

Thanks for the introduction of RabbitMQ with a Java client. Here is my feedback:

**Positive**
* The demo topic is attractive to me and it is simple to understand 
* A relevant GitHub repo is given and is easy to operate on
* Clear voice, good timing

**Possible Improvement**
* Maybe more further discussion of the advantages of message brokering within DevOps

**Comment by andrebrogard** [comment]
Feedback:

Thanks for this video!

## Strengths
* Interesting topic
* Slow and steady voiceover. Great for a demo.
* Great explanations of the actions and what is happening

## To improve 
* Could explain more the 'work queue pattern' and expand on the motivation of the demo.
* Could benefit from more visuals for clarifications. 
* Text size and only show what is important.

**Comment by andreaskth** [comment]
Hi! Thanks for the video on a very interesting topic I’ve been meaning to learn about (RabbitMQ). First of all, I think you’re in a very good position because your content seems to fit the 3 minute limit nicely, so much so that you even have time for pausing to think of what words to say and so on (which was not the case for most other demo:s I’ve seen, which had to speed up the voice-over).

If you have the time until Friday, you could try to cut out some of these silences to save ~5 seconds and that time could be used to add more content. Perhaps a quick comment on what RabbitMQ is (or message brokering in general).

In fact, I found that the video was watchable at 1.25x speed, so you could even speed it up a little to save yet more time if you want to add more information (though apart from what I mention below I don’t think there are any obvious points I felt were missing).

**Anyways, some specific points I thought about while watching the video were:**

Could briefly mention what the ’work queue’ pattern is.

At two places in the video (maybe more), you have a window flashing by for like 1 second, which is a bit confusing (roughly at 00:54 and 1:17).

The setup with three terminal windows where we can see everything at once was a very nice touch (as opposed to clicking between different ones when sending and receiving messages, which probably would have been confusing).

Very good and clear showcase at the end of the video of everything working!

Overall I don't have much to critique because this was a well-executed video. Good job and good luck with the final hand-in on Friday!

**Comment by iriediese** [comment]
Thank you all for your comments. have read everything and I am very grateful for the feedback we've received.
After carefully re-examining the entire demo, we have decided to re-record the audio track. The pace is now slightly quicker, and there is a bit more information presented in the demo. Consequently, we also re-wrote the subtitles and added more information to the video description. New link: https://youtu.be/_k3Ne-3Wpk4
